test-utils/build.gradle

```diff
@@ -1,6 +1,12 @@
 dependencies {
+    implementation project(':metrics')
+
     compile group: 'commons-io', name: 'commons-io', version: '2.6'
     compile group: 'de.heikoseeberger', name: "akka-http-play-json_$scalaVersion", version: '1.18.1'
     compile group: 'org.scala-lang.modules', name: "scala-async_$scalaVersion", version: '0.9.7'
     compile "org.scalatest:scalatest_$scalaVersion:3.0.5"
+    compile "org.mockito:mockito-all:1.10.19"
```

Comment:
What do you think about avoiding using mockito and deciding for a more scala-friendly library instead?

Reply:
Fair point. I do have our own little wrapper but wonder if we should not migrate.